BrainCloudClient/Assets/BrainCloud/Client/BrainCloud/ServerCallback.cs

```diff
@@ -42,6 +42,25 @@ public void OnErrorCallback(int statusCode, int reasonCode, string statusMessage
             }
         }
 
+        public void AddCallbacks(ServerCallback in_callback)
+        {
+            //Adding Successes
```

Comment:
m_fnSuccessCallback += in_callback.m_fnSuccessCallback ? Wouldn't this offer the same thing? Appended the callbacks to the original ? 
No need to grab the invocation list and add it to it?

Same goes for the failure callback

Reply:
I did it this way due to this compile error: 
The event 'm_fnSuccessCallback' can only appear on the left hand side of += or -= (except when used from within the class 'BrainCloud.ServerCallback')